src/utils.ts

```diff
@@ -1,13 +1,21 @@
 import { MCEvent } from '@managed-components/types';
 import * as crypto from 'crypto';
 
-export const getDomainHash = (domain: string) => {
-  return crypto
-    .createHash('sha1')
-    .update(domain)
-    .digest('hex')
-    .slice(0, 4);
-}
+export const getDomainHash = str => {
```

Comment:
@jonnyparris, decided not to use the "crypto" module at all, as it is enough to just generate a four-character string that is unique for each domain name in this case